install/windows/docker-ee.md

```diff
@@ -203,13 +204,13 @@ Start-Service Docker
 
 ## What to know before you install
```

Comment:
This link is not helpful at all in the install process  - it needs to be a system requirements page.  As a customer I would expect that link to actually tell me about pre-install configuration and system requirements. 
This one gives no useable information, it's a link to other links and it's in a confusing location in the doc near Next steps.